components/bl-image-list/src/index.js

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { ImageList, ImageListItem, ImageListItemBar } from './lib/mui';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const POS_BELOW = 'below';
+const TYPE_QUILTED = 'quilted';
+const IMAGE_HEIGHT_REDUCTION_FACTOR = 0.7;
+
+export default function ImageListComponent({ component, eventHandlers, elRef }) {
+  const {
+    style, classList, display, itemData, listVariant, cols, rowHeight,
+    gap, showTitleBar, titleBarPosition, titleBarHeight,
+  } = component;
+
+  const [items, setItems] = useState([]);
+  const imageStyle = useCalculatedImageStyle(showTitleBar, rowHeight, titleBarHeight, titleBarPosition);
+  const isQuilted = useMemo(() => listVariant === TYPE_QUILTED, [listVariant]);
+
+  useEffect(() => {
+    setItems(itemData);
+  }, [itemData]);
+
+  const onItemClick = (event, item) => {
+    eventHandlers.onItemClick({ event, item });
+  };
+
+  const onTitleClick = (event, title, item) => {
+    eventHandlers.onTitleClick({ event, title, item });
+  };
+
+  const onImageClick = (event, img, item) => {
+    eventHandlers.onImageClick({ event, img, item });
+  };
```

Comment:
use at the beginning

`  const { onItemClick, onTitleClick, onImageClick } = eventHandlers;`

and then we can remove it